installer/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/SaaSBoostInstall.java

```diff
@@ -15,1678 +15,1109 @@
-//            if (null != envName && envName.length() <= 10 && envName.equals(envName.toLowerCase())) {
-              if (null != envName && validateEnvironmentName(envName)) {
-                isValid = true;
+    protected void updateWebApp() {
+        LOGGER.info("Perform Update of the Web Application for AWS SaaS Boost");
+        outputMessage("Build web app and copy files to S3 web bucket");
+        String webUrl = buildAndCopyWebApp();
+        outputMessage("AWS SaaS Boost Console URL is: " + webUrl);
+    }
+
+    protected void cancel() {
+        outputMessage("Cancelling.");
+        System.exit(0);
+    }
+
+    protected static Path getWorkingDirectory() {
+        Path workingDir = Paths.get("");
+        String currentDir = workingDir.toAbsolutePath().toString();
+        LOGGER.info("Current dir = {}", currentDir);
+        while (true) {
+            System.out.print("Directory path of Saas Boost download (Press Enter for '" + currentDir + "'): ");
+            String saasBoostDirectory = Keyboard.readString();
+            if (isNotBlank(saasBoostDirectory)) {
+                workingDir = Path.of(saasBoostDirectory);
             } else {
-                outputMessage("Entered value is incorrect, maximum of 10 alphanumeric characters and lowercase, please try again.");
+                workingDir = Paths.get("");
             }
-        } while (!isValid);
-
-        isValid = false;
-        do {
-            System.out.print("Enter the email address for your AWS SaaS Boost administrator: ");
-            emailAddress = input.next();
-            //outputMessage("You entered email address: " + emailAddress);
-            if (null != emailAddress && validateEmail(emailAddress)) {
-                isValid = false;
-                System.out.print("Enter the email address address again to confirm: ");
-                String confirmEmail = input.next();
-                //outputMessage("You entered email address: " + emailAddress);
-                if (null != confirmEmail && emailAddress.equals(confirmEmail)) {
-                    isValid = true;
+            if (Files.isDirectory(workingDir)) {
+                if (Files.isDirectory(workingDir.resolve("resources"))) {
+                    break;
                 } else {
-                    outputMessage("Entered value for email address does not match " + emailAddress);
+                    outputMessage("No resources directory found under " + workingDir.toAbsolutePath().toString() + ". Check path.");
                 }
             } else {
-                outputMessage("Entered value for email address is incorrect or wrong format, please try again.");
-            }
-        } while (!isValid);
-
-        String useDomain = "n";
-        isValid = false;
-        do {
-            System.out.print("Would you like to setup a domain in Route 53 as a Hosted Zone for the AWS SaaS Boost environment (y or n)? ");
```

Comment:
Why are we removing the ability to set up a Route53 Hosted Zone Domain for the newly installed environment?